src/Components/cartStore.jsx

```diff
@@ -0,0 +1,20 @@
+import { create } from 'zustand';
+
+const useCartStore = create((set) => ({
+  cart: [],
+  addToCart: (product) => set((state) => ({ cart: [...state.cart, product] })),
```

Comment:
Add an if statement to check whether the product exists or not. If the product exists then increment it count. @DarioDeLaFuente